anonymizer/tests/tests.py

```diff
@@ -77,18 +91,33 @@ def setUp(self):
         self.now = datetime.now()
         self.today = date.today()
 
-        instances = (test_models.EverythingModel(id=x,
-                                                 o1=self.o1,
-                                                 username="intial%d" % x,
-                                                 birthday=self.now + timedelta(365 * x),
-                                                 age=x,
-                                                 some_datetime=self.now,
-                                                 some_date=self.today,
-                                                 sex='X',
-                                                 price=decimal.Decimal("1.23"),
-                                                 binary=compress(x),
-                                                 )
-                     for x in xrange(1, self.NUM_ITEMS + 1))
+        instances = (test_models.EverythingModel(
+            id=x,
+            o1=self.o1,
+            username="intial%d" % x,
+            something_unique='unique%d' % x,
+            birthday=self.now + timedelta(365 * x),
+            age=x,
+            some_datetime=self.now,
+            some_date=self.today,
+            sex='X',
+            price=decimal.Decimal("1.23"),
+            binary=compress(x),
+            uuid=str(uuid.uuid4()),
```

Comment:
I think `similar_lorem` is broken for non-empty values. Most of it is tested, but `something_else` is never given a value.